primitives/src/calendar.rs

```diff
@@ -30,6 +36,38 @@ impl Default for CalendarUnit {
     }
 }
 
+impl CalendarUnit {
+    // Returns the number of seconds in the unit.
+    // Variable concepts like Month and Year use 30 and 365 days respectively.
+    const fn seconds_in(self) -> u64 {
+        const S_SEC: u64 = 1;
+        const S_MINUTE: u64 = 60;
+        const S_HOUR: u64 = S_MINUTE * 60;
+        const S_DAY: u64 = S_HOUR * 24;
+        const S_WEEK: u64 = S_DAY * 7;
+        const S_MONTH: u64 = S_DAY * 30;
+        const S_YEAR: u64 = S_DAY * 365;
```

Comment:
A problem with those constants is that they don't have unique values. I would rather prefer not mapping months and years to seconds and returning "undefined".

Reply:
OK, that makes sense.